applications/nrf_desktop/doc/power_manager.rst

```diff
@@ -3,5 +3,182 @@
+| ``hid_state``                    | ``hid_report_event``   |                                  |                      |                                    |
++----------------------------------+------------------------+                                  +----------------------+------------------------------------+
+| ``power_manager``                | ``power_down_event``   |                                  |                      |                                    |
++----------------------------------+------------------------+                                  +----------------------+------------------------------------+
+|                                  | ``wake_up_event``      |                                  |                      |                                    |
++----------------------------------+------------------------+----------------------------------+----------------------+------------------------------------+
+
+Configuration
+*************
+
+This module uses Zephyr's :ref:`zephyr:power_management` subsystem.
+It depends on :option:`CONFIG_HAS_SYS_POWER_STATE_DEEP_SLEEP_1` being enabled and selects :option:`CONFIG_DEVICE_POWER_MANAGEMENT` and :option:`CONFIG_SYS_POWER_DEEP_SLEEP_STATES`.
+
+You can enable the power manager module by selecting the ``CONFIG_DESKTOP_POWER_MANAGER_ENABLE`` option in the configuration.
+
+Timeout configuration options
+    With the ``CONFIG_DESKTOP_POWER_MANAGER_TIMEOUT`` configuration option you can set time after which the application will go to low power mode.
+    By default, the timeout is set to 120 seconds.
+
+    The ``CONFIG_DESKTOP_POWER_MANAGER_ERROR_TIMEOUT`` sets the time after which the device is turned off upon an internal error.
+
+Optional idle boolean
+    The ``CONFIG_DESKTOP_POWER_MANAGER_STAY_ON`` lets the system stay idle also when there are no active connections.
+
+For more information about configuration options, check help in the configuration tool.
+
+Implementation details
+**********************
+
+The power manager is started when the :ref:`nrf_desktop_main` module is ready (reported using ``module_state_event``).
+
+When started, it can do the following operations:
+
+* Manage `Power states`_
+* Trigger `Power down`_
+* Handle `Wake-up scenarios`_ from suspended or off states
+
+Power states
+============
+
+The application can be in the following power states:
+
+ * `Idle`_
+ * `Suspended`_
+ * `Off`_
+ * `Error`_
+
+The power manager takes control of the overall operating system power state.
+See the following section for more details about how the application states converts to the system power state.
+
+Idle
+----
+
+In this state, the board is either connected or actively looking for a new connection (advertising).
+The peripherals can be turned on, including LEDs.
+
+The application remains in this state indefinitely if the device is connected to the USB.
+In such case, the operating system will be kept in the ``SYS_POWER_STATE_ACTIVE`` state.
```

Comment:
Is this correct, @pdunaj ? The state is `Idle`, but the state define is `SYS_POWER_STATE_ACTIVE`?

Reply:
yes, idle is the application state while `SYS_POWER_STATE_ACTIVE` is operating system state